lib/Models/TableColumnStyle.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+/*global require*/
+var defaultValue = require('terriajs-cesium/Source/Core/defaultValue');
+var defined = require('terriajs-cesium/Source/Core/defined');
+var freezeObject = require('terriajs-cesium/Source/Core/freezeObject');
+
+var ColorMap = require('../Map/ColorMap');
+var serializeToJson = require('../Core/serializeToJson');
+var updateFromJson = require('../Core/updateFromJson');
+
+/**
+ * A set of properties that define how a table column should be displayed.
+ * If not set explicitly, many of these properties will be given default or guessed values elsewhere,
+ * such as in CsvCatalogItem.
+ *
+ * @alias TableColumnStyle
+ * @constructor
+ *
+ * @param {Object} [options] The values of the properties of the new instance.
```

Comment:
Should this have the values of the potential options like in other constructors that take options hashes?